scripts/upgrade

```diff
@@ -101,30 +131,36 @@ then
 	sudo python $final_path/add_sso_conf_webdav.py
 	
 	# Update seafile config
-	echo 'COMPRESS_URL = MEDIA_URL' | sudo tee -a $final_path/seahub_settings.py
-	echo "STATIC_URL = MEDIA_URL + 'assets/'" | sudo tee -a $final_path/seahub_settings.py
+	echo 'COMPRESS_URL = MEDIA_URL' | sudo tee -a $final_path/conf/seahub_settings.py
+	echo "STATIC_URL = MEDIA_URL + 'assets/'" | sudo tee -a $final_path/conf/seahub_settings.py
 	
 	# Add logrotate
 	sudo cp ../conf/logrotate /etc/logrotate.d/seafile
 	sudo sed -i "s@SEAFILE_DIR@$final_path@g" /etc/logrotate.d/seafile
 	sudo service rsyslog restart
 fi
 
+# Update seahub config for old version to version 5.0.4
+if [[ $(sudo grep -c "LOGIN_URL" $final_path/conf/seahub_settings.py) == 0 ]]
+then
+        echo "LOGIN_URL = '"$path"/accounts/login/'" | sudo tee -a $final_path/conf/seahub_settings.py
```

Comment:
I can confirm the warning by upgrading from 5.0.3 to 6.0.7
```

# yunohost app upgrade -u https://github.com/Josue-T/seafile_ynh.git seafile
Attention : ./upgrade: ligne 146: path : variable sans liaison
Succès ! seafile a été mis à jour
Succès ! La configuration de SSOwat a été générée
Succès ! Mise à jour terminée

```
After that Seafile works fine but the access from the yunohost menu (sso) seems to be broken when you are not connected to seafile (sort of loop when you click on it).
You have to use the desktop client (Right click on a folder > "See in the cloud") to authenticate directly, then the access from the SSO works again. This "path" variable missing is probably the cause of the problem.